core/src/main/resources/carbonio-files.properties

```diff
@@ -19,3 +19,6 @@ carbonio.mailbox.port=20004
 # Carbonio DocsConnector
 carbonio.docs-connector.url=127.78.0.2
 carbonio.docs-connector.port=20005
+# Carbonio Message Broker
+carbonio.message-broker.url=127.78.0.2
+carbonio.message-broker.port=20006
```

Comment:
The 20006 and the 20007 ports are used for something else in the advanced, we need to think a way to avoid conflicts every time we add a new dependency in the CE flavour